src/docs/tune-system-buffer-cache.dox

```diff
@@ -98,14 +42,7 @@ An example of configuring \c os_cache_max:
 
 @snippet ex_all.c os_cache_max configuration
 
-The \c os_cache_max configuration may not be used in combination with
-direct I/O.
-
 The \c os_cache_max configuration is based on the POSIX 1003.1 standard
 \c posix_fadvise system call and may not available on all platforms.
 
-Configuring direct I/O, \c os_cache_dirty_max or \c os_cache_max all
```

Comment:
Removing the entire paragraph seems a bit much - presumably `os_cache_dirty_max` or `os_cache_max` will still disable mmap?

Reply:
changed here [f63a1a1](https://github.com/wiredtiger/wiredtiger/pull/10809/commits/f63a1a11f4803b30a37c6ffd8a6fb7dc17520ccc)